server/athenian/api/models/metadata/jira.py

```diff
@@ -97,6 +98,30 @@ class Issue(Base):
 Issue.story_points.nullable = True
 
 
+class Comment(Base):
+    __tablename__ = "comment"
+    __table_args__ = (
+        ForeignKeyConstraint(
+            ("acc_id", "issue_id"),
+            (Issue.acc_id, Issue.id),
+            name="comment_acc_id_issue_id_fkey",
+            ondelete="CASCADE",
+        ),
+        Base.__table_args__,
+    )
```

Comment:
We don't manage this in the API, so I'd rather exclude this. As soon as we migrate metadata, the schema will be out of sync.